src/app/modules/shared-components/components/collapsible-section/switch-field/switch-field.component.ts

```diff
@@ -32,6 +32,10 @@ export class SwitchFieldComponent implements ControlValueAccessor {
   @Input() value = false;
   @Input() collapsed = false;
 
+  // If this is not empty, the switch will be disabled, and the content of this array will appear as a tooltip
```

Comment:
Is it useful to have them optional? It means `[]` has a different meaning from `undefined`.

Reply:
fixed, now the array is empty by default